src/loader.ts

```diff
@@ -4,12 +4,13 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
+const { URLSearchParams } = require("url");
```

Comment:
[This shipped in Node 10](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#browser_compatibility). We don't have an `"engines"` in our `package.json` to explicitly declare our required Node version, but Node 8 is out of official support so I think we should be fine.